sdc/tests/test_utils.py

```diff
@@ -193,3 +193,30 @@ def skip_inline(msg_or_func):
         wrapper = unittest.expectedFailure
     # wrapper = lambda f: f  # disable skipping
     return wrapper(func) if func else wrapper
+
+
+def take_k_elements(k, data):
+    random_idx = np.arange(len(data))
+    np.random.shuffle(random_idx)
+    return np.asarray(data).take(random_idx[:k])
```

Comment:
Should we set up seed here and in below function? Maybe pass seed to the functions as an argument.

Reply:
Yes, I'll add it.